mechanics/Release-Mechanics.md

```diff
@@ -0,0 +1,65 @@
+# Release Mechanics
+
+Knative core components (repo) releases from master are coordinated for the
+following reasons:
+
+1. Coordinated release makes it easier to release documentation and other
+   cross-cutting repositories.
+1. Coordinated release makes it easier for operators to install and describe
+   what release they are running.
+1. Coordinated release makes it easier to test compatibility across repos,
+   though cross-repo dependencies should generally be supported at least +/- 1
+   revision (i.e. revision N+1 of repo X should work with revision N of repo Y
+   and vice-versa).
+
+**Patch (cherrypick) releases are not coordinated across repos**; it is expected
+that each repo will perform patch releases for critical bugs and security issues
+for the last 2 releases.
+
+## Release Schedule
+
+Knative releases will be cut (automatically) every 6 weeks. Any change to this
```

Comment:
How automatic is "automatic" here? Does that mean we will have machinery/automation that literally automatically does the release, and we consider green CI meaning "ready to release", or something else?